app/src/main/java/org/projectbuendia/client/diagnostics/Troubleshooter.java

```diff
@@ -48,14 +48,14 @@ public Troubleshooter(EventBus eventBus) {
     }
 
     /**
-     * Returns true if the given issue is current active.
+     * Returns true iff the given issue is current active.
```

Comment:
I'm ok with you leaving this as `if`, it seems that `iff` isn't clear to a lot of people. Likewise, below.